packages/cardpay-subgraph/src/mappings/register-reward-program-handler.ts

```diff
@@ -0,0 +1,33 @@
+import { RewardProgramRegistrationFee } from '../../generated/RewardProgram/RegisterRewardProgramHandler';
+import { log } from '@graphprotocol/graph-ts';
+
+import { RewardProgramRegistrationPayment } from '../../generated/schema';
+import { toChecksumAddress, makePrepaidCardPayment, makeToken, makeTransaction } from '../utils';
+
+export function handleRewardProgramRegistrationFee(event: RewardProgramRegistrationFee): void {
+  makeTransaction(event);
```

Comment:
you should use `makeEOATransaction` instead. That is more inclusive (it can deal with safes)

Reply:
Got it

Reply:
Actually, this follows similar pattern to other prepaid card payments. I remember you said that creating redundant entities is fine but `makeEOATransaction` is already called in `makePrepaidCardPayment` and `makeTransaction` is already called in `makeTransaction`. So, IMO I'm better off just removing `makeTransaction`. One of the reasons for this is that I do not have the prepaid card owner surfaced in the event so an abi call in the `makePrepaidCardPayment` util will handle this.

Reply:
Sounds good

Reply:
> Also, the linting errors that you have are the “unconsumed” errors I was mentioning. You can add your abi’s to the list of abi’s we consume in the addresses module to make those linting errors go away.
> 
> Also, have you confirmed that you are able to run a full subgraph index and reach the current block without the index halting?

Yup got it. Im just gonna add some `registerRewardee` and `claim` events. And then I'll run full subgraph from genesis block to see if there are any halting errors. Thanks for checking on this.